src/main/java/io/cdap/plugin/servicenow/source/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -48,7 +49,14 @@
   private static final String FIELD_CREATED_ON = "sys_created_on";
   private static final String FIELD_UPDATED_ON = "sys_updated_on";
   private static final String OAUTH_URL_TEMPLATE = "%s/oauth_token.do";
-
+  private static final Map<Integer, Integer> retryDelay = new HashMap<Integer, Integer>() {
```

Comment:
make it immutable.

Reply:
Replaced the HashMap with ImmutableMap a752f6b05f68ae64ce9ed63c94fc7b1cc1865b12